Hub/start-selenium-grid-hub.sh

```diff
@@ -17,6 +17,11 @@ if [ ! -z "$SE_HUB_PORT" ]; then
   PORT_CONFIG="--port ${SE_HUB_PORT}"
 fi
 
+if [ ! -z "$SE_HUB_SUB_PATH" ]; then
```

Comment:
I would just use `SE_SUB_PATH`. Why?

Because we should also use it in the [Standalone](https://github.com/SeleniumHQ/docker-selenium/blob/trunk/Standalone/start-selenium-standalone.sh) and the [Router](https://github.com/SeleniumHQ/docker-selenium/blob/trunk/Router/start-selenium-grid-router.sh).

Reply:
Sounds good will do!